server/apps/demo/Dockerfile

```diff
@@ -15,7 +15,6 @@ RUN cp /root/wait-for-it/wait-for-it.sh /usr/bin/wait-for-it.sh
 
```

Comment:
```
RUN curl https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh --output /usr/bin/wait-for-it.sh
```
And remove `apt-get install -y git`

Reply:
No if we do that everybody relying on the published docker image would have the same keys, this is precisely what we want to avoid!

Reply:
I don't understand, I thought it was just a download bash script. And curl will be light  than git

Reply:
When is a RUN command executed?

```
The RUN instruction will execute any commands in a new layer on top of the current image and commit the results. The resulting committed image will be used for the next step in the Dockerfile.
```

 -> at image build time.
 
 So all people using the same image would have the same keys.
 
 All people download the demo image would have the same key.
 
 This is not what we want.